patches/base-win-embedded_i18n-generate_embedded_i18n.gni.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/base/win/embedded_i18n/generate_embedded_i18n.gni b/base/win/embedded_i18n/generate_embedded_i18n.gni
+index cf734db30123eaf79063f2e2fdf1dc0ed508be87..1f9dd8c7e5e7be4f46f1a158b62b1aff578f2e60 100644
+--- a/base/win/embedded_i18n/generate_embedded_i18n.gni
++++ b/base/win/embedded_i18n/generate_embedded_i18n.gni
+@@ -90,6 +90,10 @@ template("generate_embedded_i18n") {
+       output_rc,
+     ]
+ 
++    if (defined(invoker.sources)) {
```

Comment:
Since this was moved to a template, this is needed for the already existing sources for brave_strings.grd `patches/chrome-installer-util-BUILD.gn.patch` to force recompilation when it changes.